server/app/jobs/leader_elector_job.rb

```diff
@@ -43,11 +41,12 @@ def elect
   end
 
   def announce_election
-    MongoPubsub.publish(PUBSUB_KEY, {})
+    self.elect
```

Comment:
Subtle changes to the `LeaderElectorJob` are a little scary to review :fearful:

Reply:
Indeed, this should not be in this pr.. will cleanup.